src/meta/src/controller/cluster.rs

```diff
@@ -369,6 +370,32 @@ impl ClusterController {
             .await
             .get_worker_extra_info_by_id(worker_id)
     }
+
+    pub async fn get_back_pressure(&self) -> MetaResult<GetBackPressureResponse> {
+        let nodes = self
+            .inner
+            .read()
+            .await
+            .list_active_serving_workers()
```

Comment:
Should be streaming workers.

Reply:
You mean check like this: `worker.worker_type() != WorkerType::ComputeNode`